gradle/dependencies.gradle

```
@@ -1,9 +1,13 @@
 def groovyVer = "2.5.11"
 def spockGroovyVer = groovyVer.replaceAll(/\.\d+$/, '')
 
+configurations.all {
+  resolutionStrategy.cacheChangingModulesFor 12, 'hours'
+}
```

so we can search for these comments when merging to master
```suggestion
// this is only needed for the depends-on-otel-java-snapshot branch
configurations.all {
  resolutionStrategy.cacheChangingModulesFor 12, 'hours'
}
```